openproblems/tasks/chromatin_potential/methods/linear_regression.py

```diff
@@ -0,0 +1,169 @@
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_matrix
+import scanpy as sc
+import subprocess
+from ....tools.decorators import method
+
+
+def _chrom_limit(x, tss_size=2e5):
```

Comment:
Can you give this function a docstring please?

Reply:
Yes, I'll add docstring for each of the functions.